### PR TITLE
refactor: decouple DRP/RGD mock data into external JSON files

### PR DESCRIPTION
**Pull request by mushrafmim**
## Summary
The mock person/birth data for the DRP and RGD member data-source APIs was hardcoded inline in source, so changing a record meant editing code and rebuilding/re-pushing the Docker images. This PR moves the seed data into external `mock_data.json` files that are read **fresh on every request**, so datasets can be edited or volume-mounted at runtime without a rebuild or restart.

## Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [x] Refactoring (no functional changes)
- [ ] Performance improvement
- [ ] Other (please describe):

## Changes Made
- **DRP API (`main.bal`)**: Removed the inline `table<PersonData>` literal. Records now load from `mock_data.json` via a configurable `MOCK_DATA_PATH` (default `mock_data.json`). `loadMockPersonData()` returns `...|error` (uses `check`, not `checkpanic`) and is called **per request** inside the resource function instead of caching in a module-level table; dropped the now-unneeded `lock` block.
- **DRP `Cloud.toml`**: Added `[[container.copy.files]]` to ship `mock_data.json` into the image at `/home/ballerina/mock_data.json`.
- **DRP `mock_data.json`** (new): the 6 person records extracted from code.
- **RGD API (`mock_data.py` / `main.py`)**: Removed the inline `mock_data` dict and the import-time cache. Exposed a public `load_mock_data()` that reads/parses `mock_data.json` per call; the `get_person_info` resolver now calls it each request.
- **RGD `mock_data.json`** (new): the 5 birth records extracted from code.
- **`run-member-services.sh`**: Bind-mount `mock_data.json` read-only into both containers (`/home/ballerina/mock_data.json` for DRP, `/app/mock_data.json` for RGD); added `MSYS_NO_PATHCONV=1` to the RGD run for Windows/Git Bash. Split image tags — DRP / DRP Adapter / RGD default to `v0.1.0`, passport app stays on `latest` via a new `DIE_IMAGE_TAG`. Fixed a stale `IMAGE_PREFIX` default in the usage docs.
- **`Dependencies.toml` (DRP api + adapter)**: patch-level Ballerina dependency bumps from rebuilding; added `ballerina/io` (needed for file reads). Adapter `Cloud.toml` tag bumped to `v0.1.0`.

## Testing
- [x] I have tested this change locally
- [ ] I have added unit tests for new functionality
- [x] I have tested edge cases
- [ ] All existing tests pass
- DRP `bal build` compiles clean; RGD module imports and `load_mock_data()` returns all 5 records against the real JSON.
- Verified end-to-end via `./init.sh` with the pushed `v0.1.0` images.
- Edge case: a malformed/missing JSON now surfaces as a request-time error rather than a startup failure (intended for the fresh-read design). No automated test suite exists in these mock services.

## Checklist
- [x] My code follows the project's style guidelines
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked that there are no merge conflicts

## Related Issues
Closes #49

## Additional Notes
- **Trade-off**: the file is read and objects rebuilt on every request. Fine for mock services; it's what enables live edits without a restart.
- **Single-file bind mounts** are pinned to the original inode — editors that save via atomic rename need in-place-write configured, or the container won't see the change. Mounts are `:ro` since the services only read the data.
- The template's declared `Type/Improvement` label doesn't exist in this repo; issue #49 was filed under `enhancement`.

## Deployment Notes
- Requires the `v0.1.0` images for `drp-api`, `drp-api-adapter`, and `rgd-api` to be present in the registry (passport app remains on `latest`). Override tags via `IMAGE_TAG` / `DIE_IMAGE_TAG` env vars if needed.
- `mock_data.json` is baked into each image but can be overridden at runtime by the read-only bind mounts wired into `run-member-services.sh`.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: members/rgd/data-sources/rgd-api/mock_data.py [hunk 1]
```diff
@@ -66,192 +68,59 @@ class PersonData:
...
+
+    Reads and parses the file fresh on every call (rather than caching at import)
+    so edits to mock_data.json take effect without restarting the service.
+    """
+    data_path = os.path.join(os.path.dirname(__file__), "mock_data.json")
+    with open(data_path, "r", encoding="utf-8") as f:
+        raw = json.load(f)
+    return {
+        "birth": [_build_person(record) for record in raw["birth"]],
+    }
```
[the remark this answers is not in the document]
Comment (mushrafmim) [reply members/rgd/data-sources/rgd-api/mock_data.py]: Added, Thanks.

**Review by sthanikan2000 — APPROVED** [review]
LGTM!
Verified Running Locally.
Now the mock data is runtime configurable.
